hera_qm/xrfi.py

```diff
@@ -1526,6 +1528,8 @@ def xrfi_run(ocalfits_file, acalfits_file, model_file, data_file, history,
     # Read in data file
     uv_d = UVData()
     uv_d.read(data_file)
+    if exclude_autos:
+        uv_d.select(ant_str='cross')
```

Comment:
Hmm, would it be even better to pass in `ant_str`? Then we could do:
```suggestion
    uv_d.read(data_file, ant_str=ant_str)
```

Reply:
Done.